### PR TITLE
corrected docs around custom CA support beta v. stable

### PR DESCRIPTION
**Pull request by londoncalling**
### Proposed changes

Updated description of support for custom CA certificates in FAQ's 

### Related issue

per this user-reported issue: https://github.com/docker/for-mac/issues/343#issuecomment-253806642.

### Please take a look

@samoht @justincormack @mstanleyjones @thaJeztah 

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

File: docker-for-mac/faqs.md [hunk 1]
```diff
@@ -112,11 +112,11 @@ For a full explanation and examples, see [I want to connect to a container from
 the Mac](networking.md#i-want-to-connect-to-a-container-from-the-mac) under
 [Known Limitations, Use Cases, and
 Workarounds](networking.md#known-limitations-use-cases-and-workarounds) in the
-Networking topic.
+Networking topic.``
```
Comment (thaJeztah) [inline docker-for-mac/faqs.md]: stray back tics ?

Comment (mstanleyjones) [inline docker-for-mac/faqs.md]: What's this change about?

Comment (londoncalling) [inline docker-for-mac/faqs.md]: @mstanleyjones @thaJeztah I have no idea how that happened (stray backticks), nor what it was about :-/ good catch you guys

**Comment by londoncalling** [comment]
Okay I think this is good. Merging!

**Pull request merged**